lib/game.rb

```diff
@@ -1,56 +1,59 @@
 require_relative 'board'
-require_relative 'console'
 require_relative 'rules'
+require_relative 'human_player'
+require_relative 'dumb_player'
```

Comment:
The `dumb_player` was just an example that we used during our pairing session to prove that the polymorphism works. You can probably delete that class